examples/hyperspace/hyperspace_test.go

```diff
@@ -286,7 +286,8 @@ func TestHyperspace(t *testing.T) {
 	require.NoError(t, err)
 
 	// Wait for MsgRecvPacket on cosmos chain
-	finalStakeBal := math.NewInt(fundAmount - amountToSend + amountToReflect)
+	// finalStakeBal := math.NewInt(fundAmount - amountToSend + amountToReflect)
+	finalStakeBal := fundAmount
```

Comment:
Does this add up?

Reply:
oops! no this was a mistake. Resolved in 459c033d4fefc86b281ddae1ff9ed9e1f2792ed5